docs/source/multidc/multidc.md

```diff
@@ -0,0 +1,382 @@
+      placement:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: topology.kubernetes.io/zone
+                operator: In
+                values:
+                - us-east-1c
+              - key: scylla.scylladb.com/node-type
+                operator: In
+                values:
+                - scylla
+        tolerations:
+        - key: role
+          operator: Equal
+          value: scylla-clusters
+          effect: NoSchedule
+```
+
+Apply the manifest:
+```console
+kubectl --context "${CONTEXT_DC1}" apply --server-side -f dc1.yaml
+```
+
+Wait for to the cluster to be fully rolled out and all of its nodes to be in state UN. You can verify that by running: 
+```console
+kubectl --context "${CONTEXT_DC1}" -n scylla exec -it pod/scylla-cluster-us-east-1-a-0 -c scylla -- nodetool status
+```
+
+The expected output should look similar to the below:
+```console
+Datacenter: us-east-1
+=====================
+Status=Up/Down
+|/ State=Normal/Leaving/Joining/Moving
+--  Address      Load       Tokens       Owns    Host ID                               Rack
+UN  10.0.70.195  290 KB     256          ?       494277b9-121c-4af9-bd63-3d0a7b9305f7  c
+UN  10.0.59.24   559 KB     256          ?       a3a98e08-0dfd-4a25-a96a-c5ab2f47eb37  b
+UN  10.0.19.237  107 KB     256          ?       64b6292a-327f-4128-852a-6004039f402e  a
+```
+
+Use the below commands and their expected outputs as a reference for retrieving the PodIPs used by the cluster for inter-node communication.
+```console
+kubectl --context "${CONTEXT_DC1}" -n scylla get pod/scylla-cluster-us-east-1-a-0 -ogo-template --template '{{ .status.podIP }}'
+10.0.19.237
+
+kubectl --context "${CONTEXT_DC1}" -n scylla get pod/scylla-cluster-us-east-1-b-0 -ogo-template --template '{{ .status.podIP }}'
+10.0.59.24
+
+kubectl --context "${CONTEXT_DC1}" -n scylla get pod/scylla-cluster-us-east-1-c-0 -ogo-template --template '{{ .status.podIP }}'
+10.0.70.195
+```
+
+We are going to utilize the retrieved addresses as seeds for the other datacenter.
+
+#### Deploy the second datacenter
+
+To deploy the second datacenter, you will follow similar steps.
+
+First, create a dedicated 'scylla' namespace:
+```console
+kubectl --context "${CONTEXT_DC2}" create ns scylla
+```
+
+Replace the values in `.spec.externalSeeds` of the below manifest with the Pod IP addresses that you have retrieved earlier.
+The provided values are going to serve as initial contact points for the joining nodes of the second datacenter.
+
+For the purpose of this guide, we are assuming that the second cluster is running in `us-east-2` region and the nodes dedicated for running ScyllaDB nodes are running in zones `us-east-2a`, `us-east-2b` and `us-east-2c` correspondingly. If that is not the case, adjust the manifest accordingly.
+Having configured it, save the manifest as `dc2.yaml`:
+```yaml
```

Comment:
You can mention that name of the ScyllaCluster must be in sync for all datacenters.